### PR TITLE
remove os icons, because they look outdated

### PR DESCRIPTION
**Pull request by telephon**
Open to debate, it's also kind of funny to keep them.

### CODE REVIEW

**Comment by brianlheim** [comment]
I kind of like them :) but I agree they look outdated

**Comment by seansay** [comment]
BTW seems that the `macOS` is the new offical name for Mac

File: _includes/download.md [hunk 1]
```diff
@@ -37,9 +36,9 @@
             </td>
             <td width="32%" valign="top">
                 <h2>
-                    <img src="/images/platform_linux.gif" alt="" />
                     Linux
                 </h2>
+                 <h4>Current Version</h4>
```
Comment (brianlheim) [inline _includes/download.md]: there is an extra space here, otherwise all looks good

**Comment by brianlheim** [comment]
just resolved a spurious merge conflict - master had one formatting of the `<h4>` tag, this branch had another. should be OK now.

**Pull request merged**